### DISCUSSION TITLE
[Timeline] Investigate ability to show entire timeline range in a single view

### DISCUSSION THREAD

**Discussion by markacianfrani** [RocketCommunicationsInc/astro#871] [Feature Requests]
Is it possible to have the timeline show the entire range when the zoom is set to 0? 

Additionally, can we provide control to the developer to X number of days/hours/etc at given zoom levels?

**Comment by bjclarkLM** [comment]
We thought this was coming with Astro 7.0.0?  Did this not make that release?
